app/views/groups/index.html.erb

```diff
@@ -1,16 +1,23 @@
 <% set_page_title(t("page_titles.group_index")) %>
 
+<% if @upgrade_requested_groups.present? && (@current_user.organisation_admin? || @current_user.super_admin?)  %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= govuk_notification_banner(title_text: t("banner.default.title")) do |banner| %>
+        <% banner.with_heading(text: t("groups.index.upgrade_requests_banner_heading", count: @upgrade_requested_groups.length), tag: "h3") %>
```

Comment:
You only need the `tag` parameter if there's other non-heading content inside the notification banner:
```suggestion
        <% banner.with_heading(text: t("groups.index.upgrade_requests_banner_heading", count: @upgrade_requested_groups.length)) %>
```

Reply:
Thanks! Have updated to remove it.